client/state/selectors/index.js

```diff
@@ -55,6 +55,7 @@ export getPostLikes from './get-post-likes';
 export getRawOffsets from './get-raw-offsets';
 export getReaderFeedsForQuery from './get-reader-feeds-for-query';
 export getReaderFollowedTags from './get-reader-followed-tags';
+export getReaderFollowForBlog from './get-reader-follow-for-blog';
```

Comment:
doesn't the selector you wrote disagree with you? its called getReaderFollowForBlog but it can just as easily be grabbing rss feeds right?